app/renderer/components/main/notificationBar.js

```diff
@@ -49,6 +55,32 @@ class NotificationBar extends React.Component {
   }
 }
 
+// TODO maybe this should be merged in <NotificationBar /> and defined
+// per conditional prop such as isGlobal={conditional}
+class BraveNotificationBar extends React.Component {
```

Comment:
Please move this into a separate file

Reply:
this is very similar to `NotificationBar` and plan is to make it as the comment and merge both components together, that's why it's living there. I just avoided making a big change in this PR to do so